ofnetBgp.go

```diff
@@ -73,13 +73,26 @@ func NewOfnetBgp(agent *OfnetAgent, routerInfo []string) *OfnetBgp {
 	// Keep a reference to the agent
 	ofnetBgp.agent = agent
 
-	if len(routerInfo) > 0 {
-		ofnetBgp.vlanIntf = routerInfo[0]
-	} else {
-		log.Errorf("Error creating ofnetBgp. Missing uplink port")
-		return nil
+	uplink := &PortInfo{
+		Name:       uplinkInfo[0],
```

Comment:
Looks like I missed adding the check here. Added a similar check for vlrouter case. Will add.